code/modules/clothing/shoes/jobs.dm

```diff
@@ -58,3 +61,6 @@
 	heat_protection = SLOT_FEET
 	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
 	max_pressure_protection = FIRESUIT_MAX_PRESSURE
+	material = /decl/material/solid/metal/steel
```

Comment:
Here too, in fact they already have material set few lines above: material = /decl/material/solid/leather/synth